core/src/main/java/cz/o2/proxima/util/SerializableUtils.java

```diff
@@ -39,7 +39,6 @@
       try (ByteArrayOutputStream baos = new ByteArrayOutputStream();
           ObjectOutputStream oos = new ObjectOutputStream(baos)) {
         oos.writeObject(original);
-        oos.close();
```

Comment:
Are you sure? I think we must at least call `flush`.

Reply:
Added flush